portfolio/src/main/webapp/script.js

```diff
@@ -111,6 +111,24 @@ const sleep = async (time_ms) => (
   new Promise((resolve, reject) => setTimeout(resolve, time_ms))
 );
 
+/**
+ * Load comments and add them to the page
+ */
+const loadComments = async () => {
+  const data = await fetch('/comments');
+  const comments = await data.json();
+  console.log(comments);
```

Comment:
console.log

Reply:
removed that now 👍 https://github.com/thomasogara/step-repository/pull/23/commits/29eee52a83839ad7d2f40f930d94e322de4cd51a